src/dialogs/more-info/controls/more-info-script.ts

```diff
@@ -36,17 +98,60 @@ class MoreInfoScript extends LitElement {
     `;
   }
 
+  protected override willUpdate(
+    changedProperties: PropertyValueMap<any> | Map<PropertyKey, unknown>
+  ): void {
+    super.willUpdate(changedProperties);
+    const oldState = changedProperties.get("stateObj") as
```

Comment:
You should check if `stateObj` is in `changedProperties` first, otherwise we will set `_scriptData` on every other state change as `changedProperties.get("stateObj")` will then be `undefined`